assemble/bin/start-daemon.sh

```diff
@@ -103,6 +103,9 @@ if [[ "$SERVICE" != "tserver" || $NUM_TSERVERS -eq 1 ]]; then
    fi
    echo "Starting $SERVICE on $HOST"
 
+   OUTFILE="${ACCUMULO_LOG_DIR}/${SERVICE}_1_${LOGHOST}.out"
```

Comment:
Ahh, ok, I forgot about the other services. This is what I was thinking. You can also remove lines 86-91 as they are unnecessary.